src/components/templates/TemplateWorkflowsContent.vue

```diff
@@ -78,4 +137,21 @@ const loadWorkflow = async (id: string) => {
 :deep(.p-card-subtitle) {
   text-align: center;
 }
+
+.carousel {
+  width: 1300px;
+}
+
+/* Fallback graphics for workflows that don't have an image. */
+img.thumbnail::before {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: var(--comfy-menu-secondary-bg);
+  color: var(--fg-color);
+  content: '🗎';
```

Comment:
If you want, you can use same file icon as other components:

```css
    font-family: primeicons, sans-serif;
    content: '\e958';
```

The mappings are in node_modules/primeicons/primeicons.css.